spec/presenters/section_presenter_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+describe SectionPresenter do
+  let(:section) { double(:section, body: '', attributes: {}, attachments: []) }
+  subject { described_class.new(section) }
+
+  [:slug, :title, :summary, :valid?, :errors].each do |method|
+    it "delegates #{method} to the section" do
+      expect(section).to receive(method)
+      subject.send(method)
+    end
+  end
+
+  describe '#body' do
+    context 'a section with a footnotes div' do
+      before do
+        allow(section).to receive(:body).and_return('<div class="footnotes">')
```

Comment:
Minor: Maybe this should include the closing `div` tag like the output...? I was a bit confused why it had been added. Alternatively if it really is being added and that's important behaviour, perhaps there should be a separate example.